{"uuid": "3816e609-5e1c-4d25-b32b-18c625e626d4", "children": ["f48ce4d0-32e8-4703-898d-1154fafaaa29", "422b7fc9-e10c-4a90-90cc-c53479eea285", "74fb2048-de10-4fa5-8d92-8b8b563a439b", "9af2a76c-5d2c-4b91-8bd5-39905551f8df", "5f43c6f6-f6f2-477c-89c6-4cc1b668eef1", "f1b0013e-af22-4bcf-b56d-15f0584ae78a", "091e8836-d1bd-4947-9714-939a97d9cb70", "2bfc4a4e-5300-4f85-9967-79f953ce745b"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.InvalidArgumentException: Message: invalid argument: cannot parse capability: goog:chromeOptions\nfrom invalid argument: unrecognized chrome option: prefs\nStacktrace:\n\tGetHandleVerifier [0x0xc73783+63299]\n\tGetHandleVerifier [0x0xc737c4+63364]\n\t(No symbol) [0x0xaa1113]\n\t(No symbol) [0x0xac87dc]\n\t(No symbol) [0x0xaca0c0]\n\t(No symbol) [0x0xac524d]\n\t(No symbol) [0x0xb18c62]\n\t(No symbol) [0x0xb18746]\n\t(No symbol) [0x0xb19eaf]\n\t(No symbol) [0x0xb19c5a]\n\t(No symbol) [0x0xb0e376]\n\t(No symbol) [0x0xadd6e0]\n\t(No symbol) [0x0xade544]\n\tGetHandleVerifier [0x0xece073+2531379]\n\tGetHandleVerifier [0x0xec9372+2511666]\n\tGetHandleVerifier [0x0xc99efa+220858]\n\tGetHandleVerifier [0x0xc8a548+156936]\n\tGetHandleVerifier [0x0xc90c7d+183357]\n\tGetHandleVerifier [0x0xc7b6e8+95912]\n\tGetHandleVerifier [0x0xc7b890+96336]\n\tGetHandleVerifier [0x0xc6666a+9770]\n\tBaseThreadInitThunk [0x0x7608fcc9+25]\n\tRtlGetAppContainerNamedObjectPath [0x0x77c882ae+286]\n\tRtlGetAppContainerNamedObjectPath [0x0x77c8827e+238]\n\n", "trace": "  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 906, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"E:\\Example\\Notes\\PlanSource_Test_Automation\\test_pages\\conftest.py\", line 33, in driver\n    driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()), options=options)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 66, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 250, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 342, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Pawan.Pawar\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1750163677076, "stop": 1750163681457}], "start": 1750163677076, "stop": 1750163682236}